src/styles/cards.scss

```diff
@@ -0,0 +1,172 @@
+.page {
+
+  &__catalog {
+    display: flex;
+    justify-content: center;
+    margin-bottom: 72px;
+  }
+}
+
+.catalog {
+
+  &__container {
+    display: grid;
+    margin-top: 50px;
+    column-gap: 48px;
+    row-gap: 46px;
+
+    @media (min-width: 488px) {
+      & {
+        grid-template-columns: repeat(2, 1fr);
+      }
+    }
+
+    @media (min-width: 768px) {
+      & {
+        grid-template-columns: repeat(3, 1fr);
+      }
+    }
+
+    @media (min-width: 1024px) {
+      & {
+        grid-template-columns: repeat(4, 1fr);
+      }
+    }
+
+    @media (max-width: 488px) {
+      & {
+        grid-template-columns: 1fr;
+      }
+    }
+  }
+}
+
+.card {
+  display: flex;
+  max-width: 200px;
+  flex-direction: column;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px 16px;
+  transition: transform 0.3s ease 0s;
+
+  &__image {
+    margin: 0 auto;
+    margin-bottom: 40px;
+
+    & img {
+      height: 134px;
+      max-width: 100%;
+    }
+  }
+
+  &__info {
+
+    &:not(:last-child) {
+      margin-bottom: 16px;
+    }
+  }
+
+  &__button {
+    display: flex;
+    justify-content: center;
+    text-align: center;
+    border-radius: 5px;
+    padding: 12px 0;
+    background-color: $color_blue;
+    transition: all 0.3s ease 0s;
```

Comment:
don't use it like that, because it's the same as transition: 0.3s
It's too hard for the browser
Check this
https://github.com/mate-academy/layout_catalog_hovers/blob/master/checklist.md